src/commonMain/kotlin/io/github/koalaplot/core/xygraph/XYGraph.kt

```diff
@@ -117,6 +117,10 @@ public fun <X, Y> XYGraph(
     content: @Composable XYGraphScope<X, Y>.() -> Unit
 ) {
     HoverableElementArea(modifier = modifier) {
+        // 그래프 사이즈와 마우스 좌표의 차트상 상대 좌표 추가
+        var graphSize by remember { mutableStateOf(IntSize.Zero) }
```

Comment:
This is going to cause the graph to get measured twice and will affect performance, which already is not great with plots with a lot of data points.